tables/output_tables.Rmd

```diff
@@ -214,6 +215,157 @@ write.xlsx(list_s3_table,
 
 ```
 
+### Create `Key Resources` table S4
+
+```{r table_s4}
+# Sheet 1: R packages in Docker image
+r_packages <- data.frame(installed.packages()[, c(1,3)]) %>%
+  rename(`R Package` = "Package")
+
+# Sheet 2: Python libraries in Docker image
+python_libraries <- tribble(
```

Comment:
For this one, could you read in and parse `requirements.txt`?

Reply:
Update: Don't think we're using that... so maybe we should remove it!

Reply:
This poor unheeded line in the README 😬

Reply:
With the exceptions of the following packages (makes sense these aren't there too), `Dockerfile` and `requirements.txt` _do_ match, so I think parsing this file could be the right call.

```
setuptools	46.3.0
six	1.14.0
wheel 	0.34.2
```

This README line was (contrary to a previously deleted comment..) _definitely_ heeded.
> When adding a new package, make sure that all dependencies are also added; every package should appear with a specified version both in the Dockerfile and requirements.txt